go.mod

```diff
@@ -19,10 +19,13 @@
 
 module github.com/G-Research/yunikorn-core
 
-go 1.21
+go 1.22
+
+toolchain go1.23.2
```

Comment:
Do we want to require 1.23 for working in this repo? (I'm fine with doing that, just wondering if this is intentional.)

Reply:
Updated :relaxed: